src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -65,13 +65,58 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
       val name = GroupName("a name")
       val groupCreateResult = sdk.groupCreate(GroupCreateOpts(validGroupId, name)).attempt.unsafeRunSync.value
 
-      groupCreateResult.id.id.length shouldBe 36
+      groupCreateResult.id.id shouldBe validGroupId.id
       groupCreateResult.name.get.name shouldBe name.name
       groupCreateResult.isAdmin shouldBe true
       groupCreateResult.isMember shouldBe true
       groupCreateResult.created should not be null
       groupCreateResult.lastUpdated shouldBe groupCreateResult.created
-      groupCreateResult.needsRotation shouldBe Some(false)
+      groupCreateResult.needsRotation.value shouldBe false
+    }
+  }
+
+  "Group Add Members" should {
+    "Fail for nonexistent GroupId" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val maybeAddMembersResult = sdk.groupAddMembers(GroupId("kumquat"), Nil).attempt.unsafeRunSync
+      maybeAddMembersResult.isLeft shouldBe true
```

Comment:
You can also do `maybeAddMembersResult shouldBe 'left`

Reply:
I find this way to be more easily understood, and the `'left` syntax will be going away in the future. But let me know if you really want me to change it.